hack/e2e-internal/e2e-status.sh

```diff
@@ -58,7 +58,7 @@ until [[ ${all_running} == 1 ]]; do
     echo "All pods never 'Running' in time." >&2
     exit 1
   fi
-  statuses=($(${KUBECTL} get pods --template='{{range.items}}{{.currentState.status}} {{end}}'))
+  statuses=($(${KUBECTL} get pods --template='{{range.items}}{{.currentState.status}} {{end}}' --api-version=v1beta1))
```

Comment:
Was this hard to convert to v1beta3? .status.phase doesn't work? If non-trivial, we could leave to later, but I'd rather just clean these up now rather than leave a lot of v1beta1 uses in the code.